# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Parent  d1e6f52aaedfca09fd372f5d0267f876f67d172a
Misc: Disable OTR via chat.otr.enable.

diff --git a/mail/app/profile/all-thunderbird.js b/mail/app/profile/all-thunderbird.js
--- a/mail/app/profile/all-thunderbird.js
+++ b/mail/app/profile/all-thunderbird.js
@@ -839,17 +839,18 @@ pref("mail.chat.show_desktop_notificatio
 // 2 == No info (fill dummy values).
 pref("mail.chat.notification_info", 0);
 pref("mail.chat.play_sound", true);
 // 0 == default system sound, 1 == user specified wav
 pref("mail.chat.play_sound.type", 0);
 // if sound is user specified, this needs to be a file url
 pref("mail.chat.play_sound.url", "");
 // Enable/Disable support for OTR chat encryption.
-pref("chat.otr.enable", true);
+// Betterbird doesn't currently build the OTR library, so no point enabling this. 
+pref("chat.otr.enable", false);
 // Default values for chat account prefs.
 pref("chat.otr.default.requireEncryption", false);
 pref("chat.otr.default.verifyNudge", true);
 pref("chat.otr.default.allowMsgLog", true);
 
 // BigFiles
 pref("mail.cloud_files.enabled", true);
 pref("mail.cloud_files.learn_more_url", "https://support.thunderbird.net/kb/filelink-large-attachments");
